src/pages/Inquiry/pagination.tsx

```diff
@@ -0,0 +1,64 @@
+import { ReactComponent as ArrowLeft } from 'assets/svg/inquiry/arrow-left.svg';
+import { ReactComponent as ArrowRight } from 'assets/svg/inquiry/arrow-right.svg';
+import { ReactComponent as DoubleArrowLeft } from 'assets/svg/inquiry/double-arrow-left.svg';
+import { ReactComponent as DoubleArrowRight } from 'assets/svg/inquiry/double-arrow-right.svg';
+import { Dispatch, SetStateAction } from 'react';
+import styles from './Inquiry.module.scss';
+
+interface Props {
+  totalPage: number;
+  setPage: Dispatch<SetStateAction<number>>;
+  page: number;
+}
```

Comment:
보니까 타입정의를 위해서 react를 import해서 `Dispatch, SetStateAction` 을 가지고 온 것 같네요~!
큰 건 아니지만 다음과 같은 방법도 생각해 볼 수 있어요~!
각 방법이 가지는 장점이 뭔지를 한 번 생각해보고 바꾸고 싶은 마음이 들었다면..? 바꿔 보는 것도 좋을 것 같아요~!

1. import를 type으로 해오기
- import를 할 때 type으로 가지고 올 수 있다는 사실! 알고 있었나요? 과연 type 붙이면 어떤 일이 발생할까요?
```
import type {Dispatch, SetStateAction} from 'react';
```

2. React에서 직접 꺼내오기
- 이 경우에는 react를 굳이 import 하지 않아도 된답니다~!
```suggestion
import styles from './Inquiry.module.scss';

interface Props {
  totalPage: number;
  setPage: React.Dispatch<React.SetStateAction<number>>;
  page: number;
}
```

3. 그냥 그대로 쓰기

+add)

약간 추가적으로 쓰고 싶은 팁(?)인데, 가독성을 위해서 `Dispatch<SetStateAction<T>>`을 따로 타입을 정의해서 사용하기도 하더라구여

```tsx
//이런 느낌..?
type Dispatcher<S> = Dispatch<SetStateAction<S>>
```

근데 이건 공통적으로 적용되어야 될 문제 같아서 이렇게도 하는구나~ 하고 넘어가 주세요~!

Reply:
오옹 type을 import 하는 방법이 여러가지였군요..!!! React에서 직접꺼내오는 방법이 코드가 더 간결해지는 것 같아 수정했습니다!! 
d781c095643ab1cec06f893863fd7e7af336140a